### PR TITLE
fix: Fix for the embedded case

### PR DESCRIPTION
**Pull request by sagar-qa007**
## Description
Fix the case for embedded case

Fixes #`Issue Number`  
_or_  
Fixes `Issue URL`
> [!WARNING]  
> _If no issue exists, please create an issue first, and check with the maintainers if the issue is valid._

## Automation

/ok-to-test tags="@tag.Settings"

### :mag: Cypress test results

> [!TIP]
> 🟢 🟢 🟢 All cypress tests have passed! 🎉 🎉 🎉
> Workflow run: <https://github.com/appsmithorg/appsmith/actions/runs/12311687620>
> Commit: b4e86b07be4154330ec77a7716d1ef9d29629cea
> <a href="https://internal.appsmith.com/app/cypress-dashboard/rundetails-65890b3c81d7400d08fa9ee5?branch=master&workflowId=12311687620&attempt=1" target="_blank">Cypress dashboard</a>.
> Tags: `@tag.Settings`
> Spec:
> <hr>Fri, 13 Dec 2024 08:14:04 UTC

## Communication
Should the DevRel and Marketing teams inform users about this change?
- [ ] Yes
- [ ] No

## Summary by CodeRabbit

- **New Features**
	- Introduced a new "copy application URL" button for easier URL management.
  
- **Bug Fixes**
	- Enhanced timeout durations for server restart notifications to improve reliability.

- **Improvements**
	- Modified test setup for better navigation and application creation process.
	- Updated navigation to wait for the page header before proceeding, improving test flow.

### CODE REVIEW

File: app/client/cypress/support/AdminSettingsCommands.js [hunk 1]
```diff
@@ -79,8 +79,8 @@ Cypress.Commands.add("waitForServerRestart", () => {
   // cy.waitUntil(() => !Cypress.$(adminSettings.restartNotice).length, {
   //   timeout: 180000,
   // });
-  cy.get(adminSettings.restartNotice, { timeout: 300000 }).should("not.exist");
-  cy.get(adminSettings.appsmithStarting, { timeout: 300000 }).should(
+  cy.get(adminSettings.restartNotice, { timeout: 600000 }).should("not.exist");
+  cy.get(adminSettings.appsmithStarting, { timeout: 600000 }).should(
     "not.exist",
```
[the remark this answers is not in the document]
Comment (sagar-qa007) [reply app/client/cypress/support/AdminSettingsCommands.js:84]: tell me whole command without plus

**Review by NandanAnantharamu — APPROVED** [review]
LGTM

**Pull request merged**